reference_designs/ipu-xrt/vision_pipelines/edge_detect/Makefile

```diff
@@ -21,13 +21,19 @@ EDGEDETECT_HEIGHT = 1080
 
 targetname = edgeDetect
 
-all: build/final_${EDGEDETECT_WIDTH}.xclbin
+default: peano
+
+all: peano xchesscc peanoxbridge
 
 mlir: build/aie2_lineBased_8b_${EDGEDETECT_WIDTH}.mlir
+peano: build/final_${EDGEDETECT_WIDTH}.xclbin
+xchesscc: build_xchesscc/final_${EDGEDETECT_WIDTH}.xclbin
+peanoxbridge: build_peanoxbridge/final_${EDGEDETECT_WIDTH}.xclbin
 
+PEANO = /root/acdc/build/install/peano/bin/clang++
```

Comment:
Can you get rid of this absolute path?

Reply:
ah yeah, good point.  Really all of those makefiles need to go.